packages/account/src/Sections/Security/TwoFactorAuthentication/digit-form.jsx

```diff
@@ -1,51 +1,58 @@
-import React from 'react';
+import React, { useEffect, useRef, useState } from 'react';
```

Comment:
follow Deriv-app code standard here, you don't need to decouple hooks.
just use React.useEffect...